task/grype/0.1/grype.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: grype
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Security
+    tekton.dev/tags: CLI, grype
+    tekton.dev/displayName: "grype"
+    tekton.dev/platforms: "linux/amd64,linux/arm64"
```

Comment:
Does the platforms stuff work if there's a sha256? does the image cover both architectures or are there separate images per architecture for the same tag? e.g. would have separate shas

Reply:
>Does the platforms stuff work if there's a sha256? 
Yes because this digest points to its image index (manifests list)